src/buildercore/trop.py

```diff
@@ -132,6 +132,16 @@ def rds_security(context):
                           ingress_ports,
                           "RDS DB security group")
 
+def docdb_security(context):
```

Comment:
Is a unit test needed for this function? I can't see one for the `rds_security` function so I'm assuming just updating the template in `test_docdb` is sufficient.

Reply:
it's output will be covered in the test you've already touched upon